src/toil/common.py

```diff
@@ -446,8 +448,13 @@ def _addOptionFn(*name, **kwargs):
                      'LSF, or Slurm')
     addOptionFn("--maxLogFileSize", dest="maxLogFileSize", default=None,
                       help=("The maximum size of a job log file to keep (in bytes), log files larger "
-                            "than this will be truncated to the last X bytes. Default is 50 "
-                            "kilobytes, default=%s" % config.maxLogFileSize))
+                            "than this will be truncated to the last X bytes. Setting this option to a negative "
+                            "value will prevent any truncation. Default=%s" % config.maxLogFileSize))
```

Comment:
Apply bytes2human to default value? Should print nicely with a default that is a power of two.